packages/eslint-plugin/lib/rules/no-useless-constructor.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @fileoverview Disallow unnecessary constructors
+ * @author Armano <https://github.com/armano2>
+ */
+'use strict';
+
+const baseRule = require('eslint/lib/rules/no-useless-constructor');
+const util = require('../util');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = Object.assign({}, baseRule, {
+  meta: {
```

Comment:
I would class this as recommended: error

Reply:
we should not enable rules without major release